packages/table/src/common/grid.ts

```diff
@@ -308,6 +308,11 @@ export class Grid {
         const cardinality = Regions.getRegionCardinality(region);
         switch (cardinality) {
             case RegionCardinality.CELLS: {
+                // if the region is outside the bounds of the table, don't display it
+                if (this.isGhostIndex(region.rows[0], region.cols[0])
```

Comment:
nit: Since these array values are being accessed twice now, I'd consider pulling them out into their own named variables, e.g.:

```
const [rowStart, rowEnd] = region.rows;
const [colStart, colEnd] = region.cols;
```